### PR TITLE
Create Ritisha542.json

### PR DESCRIPTION
**Pull request by Ritisha542**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6460"></a>

### CODE REVIEW

File: data/Ritisha542.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Ritisha Sen",
+  "type": "personal",
+  "bio": "CS Student",
+  "links": [
+    {
+      "name":"Follow me on Github",
+      "url": "https://github.com/Ritisha542",
+      "icon":"FaGithub"
+    },
```
Comment (jaydeepkhatri) [inline data/Ritisha542.json:10]: Hi @Ritisha542,
Thank you for contributing to Linkfree.

It seems that the JSON is invalid.
You need to close the links array.

You can test you JSON code in playgroud before making a PR.
-> https://linkfree.io/playground

Thanks :)

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]

File: data/Ritisha542.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+}
```
Comment (SaraJaoude) [inline data/Ritisha542.json:11]:
```suggestion
]
}
```

File: data/Ritisha542.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Ritisha Sen",
+  "type": "personal",
+  "bio": "CS Student",
+  "links": [
+    {
+      "name":"Follow me on Github",
+      "url": "https://github.com/Ritisha542",
+      "icon":"FaGithub"
+    },
```
Comment (SaraJaoude) [inline data/Ritisha542.json:10]:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
I have corrected the JSON so we can merge the Pull Request

**Pull request merged**